### PR TITLE
LPS-150275 Add tests for SpacingBox

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/Tooltip.test.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+			<button ref={handleRef} type="button">
+				Button
+			</button>
+		</>
+	);
+}
+
+describe('Tooltip', () => {
+	afterEach(() => {
+		cleanup();
```
Comment (veroglez) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/Tooltip.test.js:59]: `cleanup()` with the latest version of React Testing Library is not necessary.

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/SpacingBox.test.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+			{label: '0', value: '0'},
+			{label: '10', value: '10'},
+		]}
+		value={value}
+	/>
+);
+
+describe('SpacingBox', () => {
+	afterEach(() => {
+		cleanup();
```
Comment (veroglez) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/SpacingBox.test.js:35]: Same here

**Comment by veroglez** [comment]
Looks very nice! I just added some comments about the cleanup in the tests.

**Comment by p2kmgcl** [comment]
Done! :smile:

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
ci:stop

**Comment by p2kmgcl** [comment]
(I am going to add a test for the other story)

**Comment by p2kmgcl** [comment]
ci:forward

**Pull request closed without merging**